### PR TITLE
controller: add a version string

### PR DESCRIPTION
**Pull request by gireeshpunathil**
To be consumed directly, or by appsody CLI, for it to augment with its own version information.

Refs: https://github.com/appsody/appsody/issues/101

rationale for the version `0.6.0`
 - used a semantic versioning: major, minor, patch
 - looked the commit history of main.go and accounted for 6 backward compatible changes.

feel free to propose amendments! cc @kylegc @kewegner @chilanti

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 )
 
 var (
-	VERSION = "vlatest"
+	VERSION = "0.6.0"
```
Comment (kylegc) [inline main.go]: Please undo this change. We inject the version number in the build process.

Comment (gireeshpunathil) [reply main.go]: @kylegc - reverted, thanks; didn't know about this.

**Comment by kylegc** [comment]
@kewegner you are correct. For the appsody CLI, we plan to run this new version command inside docker in order get it on any platform.
